backend/routes/api/uploadRoutes.js

```diff
@@ -4,7 +4,7 @@ var uploadService = require('../../services/uploadService'),
 module.exports = function (app) {
 
     app.post('/postMessage', function (req, res, next) {

```

Comment:
path should be /api/entityName/otherData.   + /api/entityName - move to baseUrl variable